Makefile

```diff
@@ -72,11 +72,13 @@ proto: ## Rebuild protobuf files
 	go run scripts/protobuf.go generate
 
 .PHONY: build-packages
-build-packages: build-packages-race build-packages-normal ## Test docker images locally
+build-packages: build-packages-race build-packages-normal build-npm ## Test docker images locally
 build-packages-race:
 	go install -v ./...
 build-packages-normal:
 	go install -v -race ./...
+build-npm:
+	cd web/satellite && npm install
```

Comment:
maybe it makes sense to use `npm ci` command. It could speed up this process ( up to 100% )

`npm ci` ignores the package.json of the package and installs the modules based on the package-lock.json.

Reply:
oh, cool, wasn't aware of `npm ci`.